lib/portlayer/exec/container.go

```diff
@@ -166,7 +166,8 @@ func (c *Container) Commit(ctx context.Context, sess *session.Session, h *Handle
 // Start starts a container vm with the given params
 func (c *Container) Start(ctx context.Context) error {
 	defer trace.End(trace.Begin("Container.Start"))
-	//no need to grab the lock, there is no state change to the container
```

Comment:
hasan and I talked about the possibility of something in that container getting commit and changing  mid call due to a reconfigure. or if c.vm changed.